flight/extract_gps.py

```diff
@@ -0,0 +1,43 @@
+from collections import namedtuple
+import json
+
+def extract_gps():
+    with open("flight/waypoint_data.json") as waypoint_data:
+        dataFile = json.load(waypoint_data)
```

Comment:
Need to type hint variables.

Again, see: https://missourimrr.github.io/docs/python/type-annotations/full-guide.html